src/main/java/io/lettuce/core/api/reactive/RedisAclReactiveCommands.java

```diff
@@ -144,8 +144,8 @@ public interface RedisAclReactiveCommands<K, V> {
     Mono<String> aclLogReset();
 
     /**
-     * When Redis is configured to use an ACL file (with the aclfile configuration option),
-     * this command will save the currently defined ACLs from the server memory to the ACL file.
+     * When Redis is configured to use an ACL file (with the aclfile configuration option), this command will save the currently
+     * defined ACLs from the server memory to the ACL file.
```

Comment:
it was in better shape here. And what will happen if i try to put, lets say some table-like info in comment with columns,, it would start to touch some nerves quickly =)

Reply:
Same as above, merge issues, you can see that the verification in the CI fails.

I will se what happened and send another version.